packages/react-components/src/components/Switch/Switch.tsx

```diff
@@ -42,9 +49,41 @@ export const Switch: React.FC<SwitchProps> = ({
     }
   }, [on]);
 
-  const iconSize: IconSize = size === 'basic' ? 'small' : 'xsmall';
+  const getIcon = (iconSize: IconSize): JSX.Element => {
+    if (loading) {
+      return <Loader size="cover" />;
+    }
+
+    if (disabled) {
+      return (
+        <Icon
+          data-testid={`${baseClass}__disabled-icon`}
```

Comment:
Wouldn't classname be enough?

Reply:
I tried it with `className` before, but it's hard to query elements for it. Got 3 reasons for that:
1) If you want to check if the element is not in the DOM then `getElement..` returns undefined, because it can't find such element and you don't have 100% sure if you didn't mess up your test (made a typo or something)
2) Now the other method - `queryBy..` really does return information (even if element is not in the DOM), but I can't seem to use it on classNames
3) I want it to be unique so I want to use specific id for that